drawer_logic_test.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/smarty/assertions"
+)
+
+func TestLogicNoWords(t *testing.T) {
+	var w Words
+	resultChan := make(chan result)
+
+	go FakeDrawerInBrnoLogic(w, 3, resultChan)
+
+	r := <-resultChan
+	if assertions.ShouldBeError(r.Error, "words map cannot be empty")  != "" {
+		t.Fatalf(fmt.Sprintf("Expected 'words map cannot be empty' error, got '%s'", r.Error))
+	}
```

Comment:
This is weird. Either the `smarty/assertions` sucks or it is not being used the right way. The whole point of assertions is that you do not have the `if` all the conditions.
With testify it would look something like this:
```
assert.NotNil(r.Error)
```